wordpress-mobile/WordPress-iOS
# Added a conflict resolution alert to the PostList

- Added an alert to the PostList view when posts with a version conflict are selected
- The user is given two options: Discard Web, or Discard Local
- Upon selection, the appropriate version is discarded, and the editor is loaded with the version that has been retained

Fixes (in part) #13683

To test:

Create

1. Create a post using the app and save as a draft
2. Open the post, type some content and then close the app (without exiting the editor)
3. Go to the web, change the post content and publish it
4. Go back to the app, the post will have a "Local changes" label and will never update unless the user discards the changes
5. You should see now the post has a “Conflicted” state.
6. Tapping on the post with a version conflict with give two options: Discard Local, or Discard Web
7. Discard Local should remove local changes and load the editor with the latest version from the web
8. Discard Web should remove the latest web revision, and load the editor with the local changes

PR submission checklist:
